Bio/Align/emboss.py

```diff
@@ -6,8 +6,8 @@
 # package.
 """Bio.Align support for "emboss" alignment output from EMBOSS tools.
 
-This module contains a parser for the EMBOSS pairs/simple file format, for
-example from the alignret, water and needle tools.
```

Comment:
Does ``alignret`` not exist in EMBOSS anymore or something?

Reply:
I believe so, yes. I didn't find it in the current EMBOSS distribution, and google only finds it in biopython.